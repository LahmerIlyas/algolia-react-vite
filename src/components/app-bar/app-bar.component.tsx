import Typography from '../typography/typography.component'

export const AppBar = () => {
  return (
    <div className="flex flex-row justify-between align-center py-4 border-b-2 border-gray-100">
      <div className="flex flex-row align-center gap-x-8">
        <Typography as="a" variant="body" className="text-green-500">Chat with us</Typography>
        <Typography as="a" variant="body">+420 336 775 664</Typography>
        <Typography as="a" variant="body">info@freshnesecom.com</Typography>
      </div>
      <div className="flex flex-row align-center gap-x-8">
        <Typography as="a" variant="body" className="text-green-500">Blog</Typography>
        <Typography as="a" variant="body" className="text-green-500">About US</Typography>
        <Typography as="a" variant="body" className="text-green-500">Careers</Typography>
      </div>
    </div>
  )
}
