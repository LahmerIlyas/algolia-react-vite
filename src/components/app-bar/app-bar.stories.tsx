import React from 'react'
import type { ComponentMeta, ComponentStory } from '@storybook/react'

import { AppBar } from './app-bar.component'

export default {
  title: 'Example/AppBar',
  component: AppBar,
} as ComponentMeta<typeof AppBar >

const Template: ComponentStory<typeof AppBar> = () => (
  <React.Fragment>
    <AppBar/>
  </React.Fragment>
)

export const ExampleStory = Template.bind({})
ExampleStory.args = {
}
