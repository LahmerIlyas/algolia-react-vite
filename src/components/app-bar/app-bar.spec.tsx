import { describe, expect, it } from 'vitest'
import renderer from 'react-test-renderer'
import { AppBar } from './app-bar.component'

describe('Given appbar element', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<AppBar />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
