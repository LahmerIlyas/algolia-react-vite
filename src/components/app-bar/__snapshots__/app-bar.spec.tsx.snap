// Vitest Snapshot v1

exports[`Given appbar element > should renders correctly 1`] = `
<div
  className="flex flex-row justify-between align-center py-4 border-b-2 border-gray-100"
>
  <div
    className="flex flex-row align-center gap-x-8"
  >
    <a
      as="a"
      className="text-body text-green-500"
    >
      Chat with us
    </a>
    <a
      as="a"
      className="text-body"
    >
      +420 336 775 664
    </a>
    <a
      as="a"
      className="text-body"
    >
      info@freshnesecom.com
    </a>
  </div>
  <div
    className="flex flex-row align-center gap-x-8"
  >
    <a
      as="a"
      className="text-body text-green-500"
    >
      Blog
    </a>
    <a
      as="a"
      className="text-body text-green-500"
    >
      About US
    </a>
    <a
      as="a"
      className="text-body text-green-500"
    >
      Careers
    </a>
  </div>
</div>
`;
